### DISCUSSION TITLE
DateTimePicker

### DISCUSSION THREAD

**Discussion by CP3164094** [AvaloniaUI/Avalonia#12689] [Q&A]
Is there a datetimepicker available? If not, how should separate datepicker and timepicker be set so that they can be fully displayed in a smaller space!
3q!

**Comment by timunie** [comment]
You could place them in a UnifromGrid side by side or you can roll your own control for your use-case. See https://github.com/AvaloniaUI/Avalonia.Samples/tree/main/src/Avalonia.Samples/CustomControls/RatingControlSample

**Comment by timunie** [comment]
I think this is more a question than a feature request, so I'll convert it into a Q&A.
